cardano-node/src/Cardano/Tracing/Config.hs

```diff
@@ -426,6 +430,7 @@ defaultPartialTraceConfiguration =
     , pTraceTxOutbound = pure $ OnOff False
     , pTraceTxSubmissionProtocol = pure $ OnOff False
     , pTraceTxSubmission2Protocol = pure $ OnOff False
+    , pTraceGsm = pure $ OnOff False
```

Comment:
I think it makes sense to enable this tracer by default: https://github.com/IntersectMBO/cardano-node/pull/5691/#discussion_r1504367832